### DISCUSSION TITLE
Sphinx 6 support

### DISCUSSION THREAD

**Discussion by staticdev** [pradyunsg/furo#569] [Ideas] [answered]
I am trying to test Sphinx 6 beta 2 with Furo, but because of the [dependency restriction](https://github.com/pradyunsg/furo/blob/main/pyproject.toml#L20) it is not possible. Is it possible that we have an alpha/beta version without `<6` for me to try it?

I see @pradyunsg [added](https://github.com/pradyunsg/furo/commit/d955850a89310a932ecab4e84cba0c20371a6a33) this capping, are you aware already of something in 6.0 that breaks furo?

**Comment by pradyunsg** [comment, marked as answer]
This has been added on `main`. The main piece missing there was manual testing on my part. :)

**Reply by staticdev** [reply]
@pradyunsg this is great, thanks! Can we get a release with that?

**Reply by pradyunsg** [reply]
Sometime soon, yes. :)

**Reply by catarse-natural** [reply]
+1 for release

**Reply by staticdev** [reply]
@catarse-natural working great with Py 3.11 on release 2022.12.07.
